frontend/src/index.tsx

```diff
@@ -10,22 +10,20 @@ import { BowtieVersionContextProvider } from "./context/BowtieVersionContext";
 import { DragAndDrop } from "./components/DragAndDrop/DragAndDrop";
 import { ImplementationReportView } from "./components/ImplementationReportView/ImplementationReportView";
 import { MainContainer } from "./MainContainer";
-import { implementationMetadataURI } from "./data/Site";
 import {
-  ImplementationData,
   ReportData,
-  implementationFromData,
   prepareImplementationReport,
 } from "./data/parseReportData";
 import EmbedBadges from "./components/ImplementationReportView/EmbedBadges";
+import Implementation from "./data/Implementation";
 
 const fetchReportData = async (dialect: Dialect) => {
   document.title = `Bowtie - ${dialect.prettyName}`;
   return dialect.fetchReport();
 };
 
-const fetchAllReportsData = async (langImplementation: string) => {
-  document.title = `Bowtie - ${langImplementation}`;
+const fetchImplementationReportViewData = async (implementationId: string) => {
```

Comment:
One or both of these functions seem like they could be methods on `Implementation` no? Is that easy to do?